modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/exception/ExceptionHandlerManagerTestCase.java

```diff
@@ -154,6 +159,28 @@ public void handleConnectionExceptionExtensionWithCustomConnectivityError() {
     assertThat(((ConnectionException) throwable).getErrorType().get().getNamespace(), is(EXTENSION_NAMESPACE));
   }
 
+  @Test
```

Comment:
Please add an Allure annotation relating the new test to the issue that required its creation.

Like this:
`@Issue("W-10617943")`

You can also add a more human friendly description to the test with:
`@Description("Your description here")`

Reply:
Done Thanks